src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -204,6 +230,47 @@ protected override void Free()
             base.Free();
         }
 
+        protected override bool TryGetVariable(VariableIdentifier identifier, out int slot)
+        {
+            return _variableSlot.TryGetValue(identifier, out slot);
+        }
+
+        protected override int AddVariable(VariableIdentifier identifier)
+        {
+            int slot = nextVariableSlot++;
```

Comment:
>nextVariableSlot [](start = 23, length = 16)

Would it make sense to get rid of this field altogether? Can we simply look at ```_variableSlot``` to figure out the next available slot?